### PR TITLE
Add VAT number display to old version of partner page

### PR DESCRIPTION
**Pull request by mdole**
More context: this is a followup from Force #4322, which added VAT display to the newer version of the `partner` page, `partner2`. At the time, I believe we thought we didn't need to also implement it on the old version of the page (`partner`). As we were testing with a newly-created partner on staging, we discovered that at least that partner still did use `partner` instead of `partner2`. I would like to follow up by deprecating `partner`, but for now, putting up this patch.

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>The staging version of Metaphysics has breaking changes, this might not affect your build, but could mean your queries don't work the way you expect. See below for breaking changes:</td>
    </tr>
  </tbody>
</table>


- :warning: Field 'ConsignmentSubmission.edition_size' changed type from 'Int' to 'String'
- :warning: Input field 'CreateSubmissionMutationInput.edition_size' changed type from 'Int' to 'String'
- :warning: Input field 'UpdateSubmissionMutationInput.edition_size' changed type from 'Int' to 'String'
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 37180bc43c7f451762581d140e0e664e77107cee
</p>

File: src/desktop/apps/partner/client/contact.coffee [hunk 1]
```diff
@@ -25,4 +25,5 @@ module.exports = class PartnerContactView extends Backbone.View
     @$('.partner-locations').html locationStrings.join("")
 
   renderAdditionalInfo: ->
+    @$('.partner-vat-info').html "VAT ID #: #{@partner.get('vat_number')}" if @partner.get('vat_number')
```
Comment (iskounen) [inline src/desktop/apps/partner/client/contact.coffee]: We don't clear the `vat_number` field if a user changes their status to 'ineligible' or 'exempt', so we could end up showing a VAT number for either of those situations if a partner's VAT status changes. Should we also add a check for `vat_status` here?

File: src/desktop/apps/partner/test/client/contact.test.coffee [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+  describe '#renderAdditionalInfo', ->
+
+    it 'renders additional info', ->
+      view = new Contact
+        profile: new Profile fabricate 'partner_profile'
+        partner: new Partner fabricate 'partner', vat_number: "ABC123"
+        el: $('body')
+      view.renderAdditionalInfo()
+      $(view.el).html().should.containEql '<div class="partner-vat-info">VAT ID #: ABC123</div>'
```
Comment (iskounen) [inline src/desktop/apps/partner/test/client/contact.test.coffee]: Thanks for adding a test (including all the setup) for this fix! 🙏

File: src/desktop/apps/partner/client/contact.coffee (hunk 1 above)
Comment (mdole) [reply src/desktop/apps/partner/client/contact.coffee]: Good thought - I think I'd say it doesn't matter though. Since this information is publicly available, and since it's a legal requirement for us to display (and I'm not 100% sure when we can _not_ display, e.g. if they were previously registered and people made purchases from them, is there some requirement for a length of time it has to be displayed afterwards?), I feel like it's OK to continue to display it even if they are no longer collecting VAT. 

TL;DR doesn't feel like it's hurting to display in those cases and might be helping.

Comment (iskounen) [reply src/desktop/apps/partner/client/contact.coffee]: > if they were previously registered and people made purchases from them, is there some requirement for a length of time it has to be displayed afterwards?

This point especially resonates with me. I think we should figure that out, but it shouldn't hold this work up.
